src/settings.cpp

```diff
@@ -460,34 +460,28 @@ int32 IntSettingDesc::Read(const void *object) const
 }
 
 /**
- * Set the string value of a setting.
- * @param object The object the setting is to be saved in.
- * @param str The string to save.
+ * Validate the value of a setting.
+ * @param str The string to validate.
```

Comment:
As by IRC conversation, this might not be the most clear name for what this does. At least we need to extend the doxygen comment to reflect it changes the value.